models/incremental/Intermediate/int_dbt_run_results.sql

```diff
@@ -0,0 +1,47 @@
+{{ config( materialized='incremental', unique_key='command_invocation_id' ) }}
+
+with run_results as (
+    select *
+    from {{ ref('stg_dbt_run_results') }}
+),
+
+
+fields as (
+
+    select
+
+        run_results.*,
+        -- matches our dbt job names to their job ids
+        case 
+            when dbt_cloud_job_id = '5969'
+                then 'refresh pipes'
+            when dbt_cloud_job_id = '6128'
+                then 'bi hourly run, exclude optimove'
```

Comment:
this is a new thing since we're removing optimove models today, but this can just be `bi hourly run` now.
and `full daily run, optimove test` -> `full daily run`